### PR TITLE
feat: Terms and Privacy on Splash

### PR DESCRIPTION
**Pull request by corn-potage**
- Refactored Splash
- Terms and Privacy added to splash

### CODE REVIEW

**Review by jgresham — DISMISSED** [review]

File: src/renderer/Generics/redesign/Splash/Splash.tsx [hunk 1]
```diff
@@ -45,15 +45,15 @@ const Splash = ({ onClickGetStarted }: SplashProps) => {
               i18nKey="genericComponents:TermsAgreement"
               components={[
                 <Linking
-                  url="http://nicenode.xyz/terms.html"
```
Comment (corn-potage) [inline src/renderer/Generics/redesign/Splash/Splash.tsx]: I just tried accessing http://localhost:8080/terms on my local instance but it doesn't seem to work ☹️ we might need to make a /terms/index.html on the web side ?

Comment (jgresham) [reply src/renderer/Generics/redesign/Splash/Splash.tsx]: /terms works on the deploy preview hmm

Comment (corn-potage) [reply src/renderer/Generics/redesign/Splash/Splash.tsx]: I just modified the web side so that it definitely works now

Comment (jgresham) [reply src/renderer/Generics/redesign/Splash/Splash.tsx]: not sure if netlify is doing it but it is adding an extra slash `/terms/` ... I'll test if we can do both

**Pull request merged**